src/io.c

```diff
@@ -122,7 +122,7 @@ static void avifIOFileReaderDestroy(struct avifIO * io)
     avifFree(io);
 }
 
-avifIO * avifIOCreateFilePtrReader(FILE * f, avifBool closeOnDestroy)
+static avifIO * avifIOCreateFilePtrReader(FILE * f)
```

Comment:
We don't need to have avifIOCreateFilePtrReader() as a separate function. Please manually inline the only call to avifIOCreateFilePtrReader(). Thanks.

Reply:
Done.